docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md

```diff
@@ -0,0 +1,102 @@
+---
+id: aws-security-data-lake-source
+title: AWS Security Data Lake Source (Beta)
+description: This document explains how to configure the AWS Security Data Lake source setup using the Sumo logic environment.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+AWS Security Data Lake source provides a web services interface that can be used to manage data from multiple AWS sources through a single interface for multiple AWS organizations.
+
+This makes AWS log setup relatively easy. Furthermore, the data available from various AWS services is in the OCSF format, making it simple to parse and correlate across different sources.
+
+
+## Configure AWS Security Data Lake Setup
+
+
+### Step 1. Grant Sumo logic access to AWS Security Data Lake Console
+
+Granting Sumo logic access to the AWS AWS Security Data Lake source console would require access to **Sumo Account ID** and **External ID**. Find the details below:
+
+1. **Account ID**. Remember to copy and secure your Sumo Logic ID: ************, it will be needed in the further steps.
+
+2. **External ID**. The External ID is formed from your Sumo Logic region identifier. The format of your Sumo Logic account identifier is:
+   `<SumoDeployment>: <SumoAccountId>`
+   where:
+
+   * **SumoDeployment** is your Sumo Logic deployment that has to be entered in lowercase such as au, ca, de, eu, fed, in, jp, us1, or us2. To find your deployment, see Sumo Logic Endpoints by Deployment and Firewall Security.
+   * **SumoAccountId** is the Organization ID shown on your Account Overview in the Sumo Logic UI. You can access it by going to **Administration** > **Account**   >  **Account Overview**.
```

Comment:
```suggestion
1. **Account ID**. Remember to copy and secure your Sumo Logic ID: ************. It will be needed in the further steps.
2. **External ID**. The External ID is formed from your Sumo Logic region identifier. The format of your Sumo Logic account identifier is: `<SumoDeployment>: <SumoAccountId>` where:
   * **SumoDeployment** is your Sumo Logic deployment that has to be entered in lowercase such as au, ca, de, eu, fed, in, jp, us1, or us2. To find your deployment, see Sumo Logic Endpoints by Deployment and Firewall Security.
   * **SumoAccountId** is the Organization ID shown on your Account Overview in the Sumo Logic UI. You can access it by going to **Administration** > **Account**  > **Account Overview**.
```